docs/guides/browser-extension/sign-in-with-ethereum.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_label: 'Sign-In with Ethereum'
-sidebar_position: 4
+sidebar_position: 
```

Comment:
why?

This breaks the CI:

<img width="797" alt="image" src="https://github.com/lukso-network/docs/assets/477945/422a3997-559f-43df-8847-e670c55583bd">

Reply:
Should be fine now, just forgot a position number for this page